### PR TITLE
Version change to 0.5.0

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

**Review by ustulation — DISMISSED** [review]

File: rust_sodium-sys/src/init_with_rng.rs [hunk 1]
```diff
@@ -85,14 +84,16 @@ extern "C" fn buf(buf: *mut c_void, size: size_t) {
...
+/// elsewhere (e.g. via [`rust_sodium::init()`][2]) but this means that our attempt to apply this
+/// seeded RNG to libsodium has not been actioned.
 ///
 /// Each sodiumoxide function which uses the random generator in a new thread will cause a new
 /// thread-local instance of the PRNG to be created.  Each such instance will be seeded with the
 /// same value, meaning for example that two newly-spawned threads calling `box_::gen_keypair()`
 /// will generate identical keys.
+///
+/// [1]: https://download.libsodium.org/doc/advanced/custom_rng.html
+/// [2]: http://docs.maidsafe.net/rust_sodium/master/rust_sodium/fn.init.html
```
Comment (Fraser999) [inline rust_sodium-sys/src/init_with_rng.rs]: I guess this link should be `https://docs.rs/rust_sodium/*/rust_sodium/fn.init.html` now.

File: systest/Cargo.toml [hunk 2]
```diff
@@ -5,11 +5,11 @@ description = "Test that bindings match native library's interface"
 homepage = "https://maidsafe.net"
 license = "MIT OR Apache-2.0"
 name = "systest"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (Fraser999) [inline systest/Cargo.toml]: We don't normally update this version - there's no need really.

File: systest/Cargo.toml [hunk 3]
```diff
@@ -5,11 +5,11 @@ description = "Test that bindings match native library's interface"
...
 
 [dependencies]
-rust_sodium-sys = { path = "../rust_sodium-sys", version = "~0.4.0" }
-libc = "~0.2.15"
+rust_sodium-sys = { path = "../rust_sodium-sys", version = "~0.5.0" }
+libc = "~0.2.28"
 
 [build-dependencies]
 ctest = "~0.1.1"
```
Comment (Fraser999) [inline systest/Cargo.toml]: `ctest` is at 0.1.4 now.

File: Cargo.toml [hunk 4]
```diff
@@ -7,12 +7,12 @@ license = "MIT OR Apache-2.0"
 name = "rust_sodium"
 readme = "README.md"
 repository = "https://github.com/maidsafe/rust_sodium"
-version = "0.4.0"
+version = "0.5.0"
 
 [dependencies]
 libc = "~0.2.27"
```
Comment (Fraser999) [inline Cargo.toml]: Should the deps in this file not get updated too?

**Review by Fraser999 — APPROVED** [review]

**Pull request merged**